_mental_models/saying-no.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: saying-no
+name: Saying "No"
+benefit: Better Decisions
+summary: Saying "yes" to requests because you don't want to upset people can cause problems.
+prerequisites: Confidence, certainty
+---
+
+#### Description
+
+I say "yes", because I don't want to be rude to people and mostly when I say "no", I feel that person feels bad about it. There might be different cases for saying "yes", but in my case, I was thinking that saying yes to an offer, question or a request makes the thing easier.
+
+But saying some requests "yes" can cause some problems, like burn-out or feeling bad. In some cases you might be ended up with saying "yes" to a case where can't handle.
+
+So being able to say "no" to certain things can help to maintain your mental health and can provide better decision making in long-run.
```

Comment:
> So being able to say "no" to certain things can help to maintain your mental health and can provide better decision making in long-run.

So being able to say "no" to certain things can help to maintain your mental health. It can also provide better decision making in long-run.